### ISSUE TITLE
Use constants for constant values

### ISSUE THREAD

**Issue by pachonk** [Mingosio/mingos.py#8]
Values that are not intended to be changed, you should use a constant instead of a var.

Examples are:

https://github.com/Mingosio/mingos.py/blob/master/get-games.py#L27

https://github.com/Mingosio/mingos.py/blob/master/get-games.py#L161

**Comment by awurster1** [comment]
Constants cannot be created in python, only variables can. As of now, dictionaries are the best and most efficient way to manage those sets of data from what I am reading.

**Comment by pachonk** [comment]
Right, but the naming of things that are not meant to be changed/mutated is in the py style guide, ALL_CAPS_LIKE_THIS
